src/nccl_ofi_topo.c

```diff
@@ -17,13 +17,8 @@
 #include "nccl_ofi_math.h"
 #include "nccl_ofi_ofiutils.h"
 
-#if HAVE_CUDA
 static const uint8_t target_class_id = 0x03;		/* Display controller class */
 static const unsigned short target_vendor_id = 0x10de;	/* NVIDIA */
-#else
-static const uint8_t target_class_id = 0x08;		/* System peripheral */
-static const unsigned short target_vendor_id = 0x1d0f;	/* Amazon */
-#endif
 
```

Comment:
Drop the comment about Neuron devices in `is_accelerator_dev()` too then.

Reply:
Adjusted comments.